style.css

```diff
@@ -310,3 +345,64 @@ body {
   border-radius: 8px;
   color: #fff;
 }
+
+@media only screen and (min-width: 708px) {
```

Comment:
- [ ] Following the project requirements, the value for the `min-width` property should be in the range of `768px` or wider.